api/api_router.js

```diff
@@ -1,19 +1,19 @@
-const router = require('express').Router();
-const restricted = require('../middleware/restricted_middleware');
-const authRouter = require('../auth/auth_router');
-const userRouter = require('../users/users_router');
-const video_update = require('../videos/video_update');
-const videos = require('../videos/videos');
-const songs = require('../songs/songs');
+const router = require("express").Router();
+const restricted = require("../middleware/restricted_middleware");
+const authRouter = require("../auth/auth_router");
+const userRouter = require("../users/users_router");
+const video_update = require("../videos/video_update");
+const videos = require("../videos/videos");
+const songs = require("../songs/songs");
```

Comment:
These damn single vs double quotes.  I think it was some weird linter in a Mac vs. PC thing.  Been a thing the whole project.